src/main/java/seedu/agendum/logic/parser/Parser.java

```diff
@@ -170,12 +170,8 @@ private Command prepareAdd(String args) {
                 return new AddCommand(title, dateTimeMap.get(ARGS_BY));
             } else if (dateTimeMap.containsKey(ARGS_FROM) && dateTimeMap.containsKey(ARGS_TO)) {
                 return new AddCommand(title, dateTimeMap.get(ARGS_FROM), dateTimeMap.get(ARGS_TO));
-            } else if (!dateTimeMap.containsKey(ARGS_FROM) && !dateTimeMap.containsKey(ARGS_TO)
-                    && !dateTimeMap.containsKey(ARGS_BY)) {
-                return new AddCommand(title);
-            } else {
-                return new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                        AddCommand.MESSAGE_USAGE));
+            } else  {
+                return new AddCommand(args);
```

Comment:
@rachx, a quick simple fix I made up. Is this valid? Basically, if we don't fit the 2 criteria, just send the entire incoming string directly to `AddCommand(Str)`